### PR TITLE
Add Script to Build Deb Packages

### PR DESCRIPTION
**Pull request by emilengler**
I made a script to build a deb packages.
I created the script according to the installation structure of `make install`.
The only thing that needs to be done later is to collect the dependencies for Debian. At the moment there is only sqlite as a dependecy.

### CODE REVIEW

**Comment by TheWiseNoob** [comment]
Thank you! All of the dependencies should be in the Debian repositories except for likely [libclastfm](http://liblastfm.sourceforge.net/). I am going to eventually install Debian and Ubuntu in a VM to test an official .deb. Maybe sooner rather than later now.

File: makedeb.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# Create Debian control file
+echo "Package: OMP
+Version: $version
+Architecture: $arch
+Maintainer: $name <$email>
+Description: Open Music Player
+Homepage: https://www.openmusicplayer.com/
+Priority: standard
+Depends: sqlite (>=2.8.17-14)" > DEBIAN/control
```
Comment (TheWiseNoob) [inline makedeb.sh]: If you or somebody adds the rest of the dependencies and make a way to install libclastfm, I will publish this. I will also start regularly providing .debs. If not, I'll eventually test it myself and finish it. No promises when.

Comment (emilengler) [reply makedeb.sh]: What is with this ?
https://packages.debian.org/stretch/liblastfm-dev

Comment (TheWiseNoob) [reply makedeb.sh]: That's a different library. Better the libclastfm, but I would need to have qt as a dependency to use it.

Comment (emilengler) [reply makedeb.sh]: Then the only way is to include the binary of libclastfm into this project and the deb. If something isn't in the debian repos we need to ship it with the software too.

Comment (TheWiseNoob) [reply makedeb.sh]: Works for me. That's what I expected. Sorry, I should have made that more clear.

**Pull request closed without merging**